index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- Project Header -->
+    <header>
+        <!-- Header toolbar -->
+        <nav>
+            <div class="content header-toolbar-container">
+                <div class="logo-container">
+                    <div class="logo-item">
+                        <h5 class="logo">My Logo</h5>
+                    </div>
+                </div>
+                <div class="menu-container">
+                    <div class="menu-item">
+                        <div class="line-1"></div>
+                        <div class="line-2"></div>
+                        <div class="line-3"></div>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <!-- Project Headline -->
+        <div class="content headline-container">
+            <div class="headline-title">
+                <h1 class="title">I'm Nicolas Glad to see you!</h1>
+            </div>
+            <div class="description-container">
+                <p class="description-item">
+                    I'm a software developer! I can make creative UI designs, feature or website. Look through my portafolio and don't hestiate to contact me.
+                </p>
+            </div>
+            <div class="contact-links-container">
+                <h6 class="link-item"><a href="#" class="link">LET'S CONNECT</a></h6>
+            </div>
+            <div class="network-container">
+                <div class="network-items">
+                    <div class="icon twitter"></div>
+                    <div class="icon linkedlin"></div>
+                    <div class="icon medium"></div>
+                    <div class="icon github"></div>
+                    <div class="icon peace"></div>
+                </div>
```

Comment:
- [ ] Replace div with a ul tag and for each social media item use li tag that has a link tag that will redirect to your different social media profiles.